.eslintrc

```diff
@@ -13,6 +13,7 @@
     "react/jsx-filename-extension": [1, { "extensions": [".js"] }],
     "react/prefer-stateless-function": 0,
     "react/destructuring-assignment": 0,
+    "react/jsx-props-no-spreading": 0,
```

Comment:
Btw, neoplatilo by sa nam poriesit ten eslint cez `packages.json`?

Reply:
Má to nějaké výhody (fakt nevím tak se ptám)? Nebo aby to bylo na jednom místě?

Reply:
Sko som myslel to, ze by sa cely `.eslintrc` vyhodil a pouzil ten novy zapis v packages json:

```js
"eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest"
    ]
  },
```
Ale to mozeme aj neskor.

Reply:
Aha, to neznám. Jo tak klidně pak mrknu hned co to udělá.